weltyt.py

```diff
@@ -38,14 +39,17 @@ def buy_plants(plants):
         plants_price = get_resource_value("price", "Plants")
 
         new_money_amount = money_amount - amount * plants_price
-        update_resource_amount(new_money_amount, "Money")
+        if new_money_amount < 0:
+            print("There is not enough money to finish this operation.")
+        else:
```

Comment:
Needless else.
You can just end the function in if conditional:
```
if new_money_amount < 0:
            print("There is not enough money to finish this operation.")
            return # it will just end the function, without doing anything
update_resource_amount(new_money_amount, "Money")
rest_of_the_code
```